### ISSUE TITLE
[Accessibility] [QMenu] Focus jumps out of the page after pressing TAB on the last element

### ISSUE THREAD

**Issue by TotallyMehis** [quasarframework/quasar#14641]
### What happened?

Opening QMenu's list and navigating past the last element in the list with TAB causes the focus to leave the page.

Codepen example:
![repro](https://user-images.githubusercontent.com/4648654/196358943-df6d91c3-81f7-44eb-8c67-42ea4e0a8226.gif)

### What did you expect to happen?

After navigating past the last element, the menu closes and the focus is set back to the button.
OR
After navigating past the last element, the focus is set back to the button and the user can loop through the list again. (similar to QSelect)

### Reproduction URL

https://codepen.io/TotallyMehis/pen/qBYGWXQ

### How to reproduce?

1. Go to reproduction.
2. Click on the problematic menu.
3. Navigate the list using TAB.
4. On the last element, press TAB again.

The focus leaves the page. (in Codepen it leaves the sandbox area)

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

**Comment by caseybishop** [comment]
I'm also having this issue. Is a fix for this planned? Is there a suggested workaround?  To be accessible, the focus needs to move to the next focusable item on the page, or return to the trigger.

**Comment by acutus** [comment]
Ran into this as well. Running v2.14.6. In addition to accessibility, this causes somewhat unexpected behavior for power users who are used to navigating pages using keyboard instead of mouse.

**Comment by rstoenescu** [comment]
Hi All,

I have explained this a bunch of times. We already catch the focus within the window and don't let it escape. The focus jumps only within the QMenu/QDialog content and the browser address bar. The latter is a browser behavior which we cannot override. However, when the developer tools are opened, the browser will treat that the same as the address bar and the user will be able to focus on those elements too. But in production, the users won't keep the developer tools opened, so this is a moot point.
